packages/datagateway-common/src/table/cellRenderers/selectCell.component.test.tsx

```diff
@@ -72,6 +72,13 @@ describe('Select cell component', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('renders correctly when selectedRows parentSelected is true', () => {
+    const wrapper = shallow(
+      <SelectCell {...selectCellProps} parentSelected={true} />
+    );
+    expect(wrapper).toMatchSnapshot();
+  });
+
```

Comment:
Can we port this test over to `selectHeader` as well? Just for completeness :)

Reply:
Oh yes sorry ill do that now